packages/typespec-ts/test/modularUnit/scenarios/operations/operations.md

```diff
@@ -1190,3 +1190,188 @@ export async function createOrUpdateEndpoint(
   return _createOrUpdateEndpointDeserialize(result);
 }
 ```
+
+# should support POST paging generation from TypeSpec
+
+## TypeSpec
+
+```tsp
+import "@typespec/http";
+import "@azure-tools/typespec-client-generator-core";
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@service(#{
+  title: "Test Service"
+})
+namespace testService;
+
+model ListTestResult {
+  @pageItems
+  tests: Test[];
+  @nextLink
+  next: string;
+}
+    
+model Test {
+  id: string;
+}
+    
+@Legacy.nextLinkVerb("GET")
+@list
+@route("/list-get")
+@post
+op bar(): ListTestResult;
+
+@Legacy.nextLinkVerb("POST")
+@list
+@route("/list-post")
+@post
+op foo(): ListTestResult;
```

Comment:
@MaryGao Is the logic for determining post paging based on `post method &paging &nextLinkVerb===post` or `post method & paging`？